Tasks/Common/packaging-common/package.json

```diff
@@ -12,16 +12,20 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@types/ltx": "^2.8.0",
-    "@types/node": "^10.12.9",
+    "@types/ini": "1.3.30",
+    "@types/ltx": "2.8.0",
+    "@types/mocha": "5.2.6",
+    "@types/mockery": "1.4.29",
+    "@types/node": "10.12.9",
+    "@types/q": "1.5.2",
     "adm-zip": "^0.4.11",
     "azure-devops-node-api": "^6.6.0",
+    "azure-pipelines-task-lib": "2.8.0",
+    "azure-pipelines-tool-lib": "0.12.0",
```

Comment:
Is this the recommended version to use? Looks like it's published very recently(16 days ago). There is one 0.11.0 which was published 5 months ago.

Reply:
I'm not sure there is a difference between 0.11.0 and 0.12.0. 0.11.0 doesn't seem to exist in master: https://github.com/Microsoft/azure-pipelines-tool-lib/commit/5b69022c93077dbaab31accdb91e5c57b418522d#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

Reply:
We are pulling these packages from npmjs, right? https://www.npmjs.com/package/azure-pipelines-tool-lib
So, all commits added in these 5 months will be in 0.12.0 but not in 0.11.0

Reply:
After assessing the changes between 0.12.0 and 0.11.0 (which came from a branch), it seems unlikely to introduce any issues.